NebulaModel/Packets/Factory/Storage/StorageSyncSetFilterPacket.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace NebulaModel.Packets.Factory.Storage;
+
+public class StorageSyncSetFilterPacket
+{
+    public StorageSyncSetFilterPacket() { }
+
+    // @TODO: This can potentially be moved up the call stack - in UI input - for storage boxes-
+    // to avoid sending redundant data like `StorageIndex` & `StorageType` per every grid slot.
```

Comment:
Yeah, but
1. The event is by the user and is seldom triggered, so there is no need to worry about bandwidth usage.
2. There are possibilities that other mods may directly call SetFilter without going the default UI route.
3. In the future if there are other buildings that use Storage (e.g. Battlefield Analysis Base), this code can be reused without adding patch to the UI.

Reply:
Understood! Removed the comment. 👍